src/orders/utils/data.ts

```diff
@@ -300,6 +301,16 @@ export const isStockError = (
   return isQuantityLargerThanAvailable || isError;
 };
 
+export function addressToAddressInput<T>(
+  address: T & AddressFragment
+): AddressInput {
+  const { id, __typename, ...rest } = address;
+  return {
+    ...rest,
+    country: findInEnum(address.country.code, CountryCode)
+  };
+}
+
```

Comment:
Can we delete it? It does more or less the same thing as `transformAddressToAddressInput` and it's not used anywhere